src/Resources/views/Block/block_gallery.html.twig

```diff
@@ -12,63 +12,28 @@ file that was distributed with this source code.
 {% extends sonata_block.templates.block_base %}
 
 {% block block %}
-    {% if settings.format %}
-        <div class="{{ settings.class }}">
-            {% if settings.title %}
-                <h3>
-                    {% if settings.icon %}
-                        <i class="{{ settings.icon }}" aria-hidden="true"></i>
-                    {% endif %}
-                    {% if settings.translation_domain %}
-                        {{ settings.title|trans({}, settings.translation_domain) }}
-                    {% else %}
-                        {{ settings.title }}
-                    {% endif %}
-                </h3>
-            {% endif %}
-
-            {% sonata_template_box 'This is the gallery media block. Feel free to override it.' %}
-
-            <div id="carousel-{{ block.id }}" class="carousel slide"
```

Comment:
same for all of this, the library used by this block should be chosen and implemented by the end user (or maybe he wants to show the gallery with only css and html, without JS.